### ISSUE TITLE
Please update Ethereum or Mist Wallet to use the 1.7.3 version of Geth

### ISSUE THREAD

**Issue by unknown1235** [ethereum/mist#3369]
- [x ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.
Ethereum wallet is still using the 1.7.2 version of Geth.
1.7.3 is already released and maybe it will fix some of the current issues.
 Under 1.7.2 it is impossible to load, transfer tokens as the whole wallet crashes when ever you add a token to watch.

```
Version: `0.9.3`
OS & Version: windows 10
Node version: `geth 1.7.2' 
Number of blocks synchronized:  fully synced
```

**Comment by hagitarowe** [comment]
I've had my chain store corrupted due to what i can only assume was the version of geth running through wallet as it happened after accidentally starting wallet before geth.

**Comment by tobiv** [comment]
Confirm 0.9.3 downloads geth 1.7.2 even if 1.7.3 is installed, it just gets overwritten.

**Comment by unknown1235** [comment]
I tried manually downloading 1.7.3 but then the Ethereum wallet gets stuck on connecting to node.
Is there anyway to get an update on this?  My tokens are stuck and can't get it out.

**Comment by zamicol** [comment]
I've been using geth and Mist independently.

There are a lot of issues right now.

**Comment by tadanderson** [comment]
Same issue.   Cannot get Mist to switch to geth 1.7.3, it is stuck on 1.7.2.  This has wrecked my production and test chains several times.  Getting sick of having to redownload the entire chain over and over.  Light Client is worthless.  Works about 3% of the time.

**Comment by zamicol** [comment]
@tobiv 
Mist will overwrite in the `Mist/binaries` directory.  It shouldn't overwrite from the `.ethereum` directory.  

You can start geth first and Mist should connect to that client.  

However, I'm having show stopping problems no matter how I run it.

**Comment by tadanderson** [comment]
@zamicol
I am running on macOS.  I could get it to connect as you described above, but 1.7.2 still showed as being connected.  It eventually always stalled.  Stripped everything off except keystores several times.

Gave up.  Removed Mist and will going to wait for an update to Mist.  Just running geth now.  Getting stalling state syncs every now and then but still downloading.

**Comment by tobiv** [comment]
Pull request is pending: https://github.com/ethereum/mist/pull/3304

**Comment by vernon99** [comment]
Yeah, so this is pretty bad. Basically if you forget to start geth first OR if you run several networks (testnet and prodnet), Ethereum Wallet/Mist just screw up your whole chaindata as they force on you the older version. In the latter case since you can switch networks only from the wallet itself, that kicks off another geth instance with the different chain. Why both Mist and Ethererum Wallet have their own geth, why they do not respect the system-wide default geth and why they override the hell out of everything?

**Comment by renestach** [comment]
Geth is at 1.8.0 in the meantime with some major changes to the light client. If I could only update geth myself.

**Comment by renestach** [comment]
Today my installation of ethereumupdated geth to 1.8.1. If others are seeing the same, this can be closed now.

**Comment by tadanderson** [comment]
@renestach The issue is still the same.  Mist hasn’t been update and still does not show working with Geth 1.8.X.  At least on Windows 10.

**Comment by tadanderson** [comment]
@renestach Actually I take that back.  1.8.1 did successfully connect to Mist.  I was running 1.8.0.  Have not tested on my MacBook because of Swarm and space.  Although once loaded, it slows the light sync mode down and never seems to fully sync.  I guess this can be closed because I had given up on Mist and seeing this, I will probably keep it out of my dev stack until I see it actually get an update.

**Comment by vernon99** [comment]
Ideally this should be renamed, not closed - the problem with Mist/Ethereum Wallet overriding global settings, breaking your chaindata, etc is still there and it will still be a pain next time.
